package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@netzstrategen/stylelint-config",
-  "version": "1.0.0",
+  "version": "1.1.0",
```

Comment:
Also just as a note, we are using conventions based on semver.org. Since this config change is a breaking change (as lots of code violations would show up) we need to mark it as `2.0.0` instead.